### PR TITLE
Add Checkbox component

### PR DESCRIPTION
**Pull request by samshara**
Need to change styling for dark mode

### CODE REVIEW

File: src/components/Checkbox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        ...otherProps
+    } = props;
+
+    const inputId = useMemo(
+        () => randomString(16),
+        [],
+    );
+
+    const handleChange = useCallback(
+        (e) => {
```
Comment (AdityaKhatri) [inline src/components/Checkbox/index.tsx]: Don't we need to define the type of event?

Comment (tnagorra) [reply src/components/Checkbox/index.tsx]: We should.

File: src/components/Checkbox/index.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    const labelClassName = _cs(
+        'label',
+        styles.label,
+        labelClassNameFromProps,
+    );
+
+    return (
+        <label
+            htmlFor={name}
```
Comment (tnagorra) [inline src/components/Checkbox/index.tsx]: We should use a random htmlFor value not the name

File: package.json [hunk 3]
```diff
@@ -40,6 +40,7 @@
     },
     "dependencies": {
         "@babel/runtime-corejs3": "^7.10.5",
+        "@storybook/client-api": "^6.0.20",
```
Comment (tnagorra) [inline package.json]: Why was this dependency added?

**Pull request merged**